gobblin-core/src/main/java/gobblin/source/DatePartitionedAvroFileSource.java

```diff
@@ -62,57 +62,57 @@
 /**
  * Implementation of {@link gobblin.source.Source} that reads over date-partitioned Avro data.
  * This source can be regarded as the reader equivalent of {@link TimeBasedAvroWriterPartitioner}.
- * 
```

Comment:
Sorry for all of these junk whitespace changes, I couldn't figure out how to make IntelliJ not make these updates